assets/js/atomic/blocks/product-elements/title/index.ts

```diff
@@ -25,6 +27,17 @@ const blockConfig: BlockConfiguration = {
 	},
 	attributes,
 	edit,
+	supports: isFeaturePluginBuild()
+		? {
+				html: false,
+				color: {
+					background: false,
+				},
+				typography: {
+					fontSize: true,
+				},
+		  }
+		: {},
```

Comment:
You need to use the `supports` from `sharedConfig` here:
```suggestion
		: sharedConfig.supports,
```